openmetadata-ui/src/main/resources/ui/src/pages/EditConnectionFormPage/EditConnectionFormPage.component.tsx

```diff
@@ -153,7 +155,9 @@ function EditConnectionFormPage() {
           <TitleBreadcrumb titleLinks={slashedBreadcrumb} />
           <div className="form-container">
             <Typography.Title level={5}>
-              {`Edit ${serviceFQN} Service Connection`}
+              {`${t('label.edit')} ${serviceFQN} ${t('label.service')} ${t(
+                'label.connection'
+              )}`}
             </Typography.Title>
```

Comment:
it is preferable to have a full sentence instead of breaking down each word!

Reply:
Make sense, I will update it.

Reply:
Done @ShaileshParmar11